plugin.video.otaku/resources/lib/ui/control.py

```diff
@@ -26,7 +26,9 @@
 INPUT_ALPHANUM = xbmcgui.INPUT_ALPHANUM
 dataPath = TRANSLATEPATH(addonInfo('profile'))
 ADDON_PATH = __settings__.getAddonInfo('path')
-_kodiver = float(xbmcaddon.Addon('xbmc.addon').getAddonInfo('version')[:4])
+try:
+    _kodiver = float(xbmcaddon.Addon('xbmc.addon').getAddonInfo('version')[:4])
+except ValueError: pass #Avoid error while executing unit tests
 
```

Comment:
Hope we're happy to keep all these in. They're quite useful when you've got the kodi_six of kodistubs installed. Allows me to run functions directly, which speeds up testing.

Reply:
We defiantly will keep these these try/excepts in sense they are quite useful which speeds up testing so we will not get rid of these.